lms/djangoapps/instructor/management/commands/collect_student_forums_data.py

```diff
@@ -0,0 +1,54 @@
+"""
+Command to retrieve aggregated student forums data in a .csv
+"""
+import csv
+import optparse
+import os
+
+from instructor.utils import collect_student_forums_data
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
Can you group these imports according to the style guide [1]?

[1] https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order